app/components/filters/include_new_or_changed.rb

```diff
@@ -0,0 +1,21 @@
+module Filters
+  class IncludeNewOrChanged
+    MAX_UPDATE_DAYS = 7 # used to limit number of entries queried to determine if an update ahs occurred.
+
+    def call
+      ETL::Filter.new(filter_name: 'Existing/Unchanged record') do |record|
+        new(record) || changed(record)
```

Comment:
It would if this was new and edit, but the filter calls for new or edited.

Refactored to inherit from IncludeNew to allow reuse of code